core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -214,17 +230,20 @@ export class AudioBarComponent {
     if (this.autogeneratedAudioPlayerService.isPlaying()) {
       this.autogeneratedAudioPlayerService.cancel();
     } else {
-      this.autogeneratedAudioPlayerService.play(
-        this.audioTranslationManagerService
-          .getCurrentHtmlForAutogeneratedAudio(),
-        this.audioTranslationLanguageService
-          .getSpeechSynthesisLanguageCode(),
-        () => {
-          // Used to update bindings to show a silent speaker after
-          // autogenerated audio has finished playing.
+      const speechSynthesisLanguageCode = (
+        this.audioTranslationLanguageService.getSpeechSynthesisLanguageCode());
+      if (speechSynthesisLanguageCode !== null) {
```

Comment:
Should we do something when this is `null`?

Reply:
Done